db/dataset_eval.py

```diff
@@ -11,12 +11,14 @@
 STATUS_RUNNING = "running"
 STATUS_DONE = "done"
 STATUS_FAILED = "failed"
-
+# Eval locations are defined in `eval_location_type` type. See schema definition.
```

Comment:
Right. Always think twice when you copy things (comments or code). Don't duplicate content just because it's already there. Also consider if it can be implemented better (integrating both parts into 1 piece of code), or if you should change both the old and the new code to be tidier, or easier to understand